x-pack/plugins/security_solution/public/common/components/risk_score/risk_score_onboarding/translations.ts

```diff
@@ -0,0 +1,42 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0; you may not use this file except in compliance with the Elastic License
+ * 2.0.
+ */
+import { i18n } from '@kbn/i18n';
+
+export const TECHNICAL_PREVIEW = i18n.translate(
+  'xpack.securitySolution.riskScore.technicalPreviewLabel',
+  {
+    defaultMessage: 'Technical Preview',
+  }
+);
+
+export const HOST_WARNING_TITLE = i18n.translate(
+  'xpack.securitySolution.riskScore.hostsDashboardWarningPanelTitle',
+  {
+    defaultMessage: 'No host risk score data available to display',
+  }
+);
+
+export const USER_WARNING_TITLE = i18n.translate(
+  'xpack.securitySolution.riskScore.usersDashboardWarningPanelTitle',
+  {
+    defaultMessage: 'No user risk score data available to display',
+  }
+);
+
+export const HOST_WARNING_BODY = i18n.translate(
+  'xpack.securitySolution.riskScore.hostsDashboardWarningPanelBody',
+  {
+    defaultMessage: `We haven't detected any host risk score data from the hosts in your environment for the selected time range.`,
```

Comment:
we probably shouldn't say for the selected time range since the time range has no effect.
```suggestion
    defaultMessage: `We haven't detected any host risk score data from the hosts in your environment.`,
```